src/components/DeploymentOptionsGrid/DeploymentOptionsGrid.tsx

```diff
@@ -0,0 +1,116 @@
+                    <div className="card-body">
+                        <div className="icon-flex-row">
+                            <div className="icon-lockup">
+                                <BadgeWrapper
+                                    style={{
+                                        background: 'rgba(251, 79, 13, 0.1)',
+                                        color: '#F75A01',
+                                    }}
+                                >
+                                    <OpenSourceIcon />
+                                </BadgeWrapper>
+                                <h3 className="font-sans">Open source</h3>
+                                <p>Free forever</p>
+                            </div>
+                            <div className="icon-lockup">
+                                <BadgeWrapper
+                                    style={{
+                                        background: 'rgba(58, 94, 244, 0.1)',
+                                        color: '#3A5EF4',
+                                    }}
+                                >
+                                    <UpAndToTheRightIcon />
+                                </BadgeWrapper>
+                                <h3>Enterprise</h3>
+                                <p>From $2k/mo</p>
+                            </div>
+                        </div>
+                        <p>Benefits</p>
+                        <BenefitsList
+                            items={[
+                                'Data stays on your infrastructure',
+                                'Full access to production instance',
+                                'No third-party cookies',
+                            ]}
+                        />
+                        <ButtonBlock
+                            onClick={() => reportDeploymentTypeSelected(Realm.hosted, '/docs/self-host/overview')}
+                        >
+                            Select
+                        </ButtonBlock>
+                    </div>
+                    <footer>Self serve. Deploy with Docker, AWS, GCS + more</footer>
+                </div>
+                <div className="pricing-card">
+                    <h2 className="bg-blue">
+                        <CloudIcon />
+                        Hosted
+                    </h2>
+                    <div className="card-body">
+                        <div className="icon-flex-row">
+                            <div className="icon-lockup">
+                                <img src={posthogIcon} style={{ height: 36 }} className="posthog-icon" />
+                                <h3>PostHog Cloud</h3>
+                                <p>Pay per event</p>
+                            </div>
+                        </div>
+                        <p>Benefits</p>
+                        <BenefitsList
+                            items={[
+                                'Scales to billions of monthly events',
```

Comment:
How about we change that one to say "first 1M events are free" as we don't have that written anywhere else now & that's a unique benefit to cloud (self hosted one still has hosting costs) + the "pay by event" makes it sound like you have to start paying immediately.

> Do we have any autoscaling support for Scale customers yet?

Nope, but preliminary perf analysis said things scale pretty well for ingestion (https://github.com/PostHog/vpc/issues/151)

Reply:
Hey @tiina303 we don't want to lead with that because we don't want cost to be a factor in this decision as discussed in the ticket here https://github.com/PostHog/product-internal/issues/111 - feel like that's a different level of incentive when at this stage the choice should really be based on whether you need to self-host or not. THat's atleast the theory we are operating under and we can change it in the future if we feel differently.

Reply:
> Nope, but preliminary perf analysis said things scale pretty well for ingestion (PostHog/vpc#151)

That's great - @samwinslow it feels like we can then go ahed and use one of the things oriented around scaling..

"Automatically scales as you grow" seems good?